src/dbcp/transform/ballot_ready.py

```diff
@@ -166,7 +165,6 @@ def _explode_counties(raw_ballot_ready: pd.DataFrame) -> pd.DataFrame:
 
     # Drop duplicates. A later version of ballot ready data will remedy this problem.
     ballot_ready = exp_ballot_ready.drop_duplicates(subset=["county", "race_id"])
-    assert ~ballot_ready.duplicated(subset=["county", "race_id"], keep=False).any()
```

Comment:
I removed this assertion because as far as I can tell it is essentially only checking whether pandas implemented `.drop_duplicates` correctly in the preceding line